flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/AbstractCatalogView.java

```diff
@@ -43,8 +43,12 @@
 	private final Map<String, String> properties;
 	private final String comment;
 
-	public AbstractCatalogView(String originalQuery, String expandedQuery, TableSchema schema,
-			Map<String, String> properties, String comment) {
+	public AbstractCatalogView(
```

Comment:
nit: maybe we can restore the file completely.

Reply:
you are right, I should have reverted unrelated changes